### ISSUE TITLE
Update contract documentation based on in-code descriptions

### ISSUE THREAD

**Issue by Chris-Hibbert** [Agoric/documentation#220]
https://github.com/Agoric/agoric-sdk/pull/1196, which will be part of [Zoe 0.7.0](https://github.com/Agoric/agoric-sdk/pull/1143) includes updated documentation of all the existing Zoe contracts. In addition to filling in some contracts that are missing from https://agoric.com/documentation/zoe/guide/contracts, it also add substantially to the categories of information that are captured.

All of this should be transformed into a format that makes sense for the public site.

The things that I tried to capture consistently, which I think should be presented in a standardized format for the public site include:
* What details are determined by the installation step
* required parts of the proposal
* what the initial invitation can be used for
* outcome of the first offer
* If the contract is a simple two-step operation (quite common) what the response to the initial offer consists of
   * shape of the counter proposal
   * restrictions on valid proposals
* what functions are available in the publicAPI
* what the contract actually does
* also, when a timer is involved in exit criteria, that should be mentioned.

Both autoswap and mutlipoolAutoswap are complex enough that I didn't capture everything. We'll need to spend a little time filling out the details of how users interact with those contracts beyond the initial setup.

The opera ticket contract is more of an example than it is a reusable contract. I'm not sure whether it should be fit into this framework, or whether it should be described in a different way.

**Comment by Chris-Hibbert** [comment]
Autoswap and multipoolAutoswap now have updated internal documentation that should also be reflected to the documentation web site.

**Comment by Tyrosine22** [comment]
I agree that the information (and format) on the contract topics should be standardized. Before I tackle this bug though, I'll want to discuss with an SME which contracts should be documented... or maybe important contracts get the standard documentation treatment while unimportant contracts just get brief write-ups? In any case, I don't expect I'll get to this bug for quite some time.
